shared/src/formationData.ts

```diff
@@ -145,10 +142,11 @@ export interface FormationFormData extends FormationAddress {
   readonly contactPhoneNumber: string;
   readonly foreignStateOfFormation: StateNames | undefined;
   readonly foreignDateOfFormation: string | undefined; // YYYY-MM-DD
-  readonly foreignGoodStandingFile: ForeignGoodStandingFileObject | undefined;
+  readonly willPracticeLaw: "Yes" | "No" | undefined;
```

Comment:
why are we redefining this when we have the value up on line 101?
Also, I would argue we don't want to store this as "Yes" and "No" but rather as a boolean true/false. The Yes/No is how the API accepts it but since this is our domain layer, we have the freedom to choose a more appropriate representation for our type. And being able to write things like `if (willPracticeLaw)` is more in line with how we expect things to work, rather than having to do `if (willPracticeLaw === "Yes")`